apps/api/src/quantagent/api/services/dashboard_api.py

```
@@ -0,0 +1,100 @@
+                "reason": None,
+                "updated_at": snapshot.generated_at,
+            }
+            metrics_payload = {
+                "new_count": snapshot.entry_metrics.new_count,
+                "featured_count": snapshot.entry_metrics.featured_count,
+                "analyzing_count": snapshot.entry_metrics.analyzing_count,
+                "failed_or_review_count": snapshot.entry_metrics.failed_or_review_count,
+                "pending_approval_count": snapshot.entry_metrics.pending_approval_count,
+            }
+            metrics_meta = {"status": "ok", "reason": None, "updated_at": snapshot.generated_at}
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)